packages/actor-query-operation-leftjoin-nestedloop/test/ActorQueryOperationLeftJoinNestedLoop-test.ts

```diff
@@ -59,7 +59,8 @@ describe('ActorQueryOperationLeftJoinNestedLoop', () => {
         left = !left;
         return Promise.resolve({
           bindingsStream: left ? bindingStreamLeft : bindingStreamRight,
-          metadata: () => Promise.resolve({ totalItems: 3 }),
+          metadata: () => arg.operation.hasOwnProperty("rejectMetadata") && arg.operation.rejectMetadata ?
```

Comment:
Can be simplified to `arg.operation.rejectMetadata ? Promise.reject(new Error('fail')) : Promise.resolve({ totalItems: 3 })`